var/spack/repos/builtin/packages/eospac/package.py

```diff
@@ -0,0 +1,41 @@
+##############################################################################
+# Copyright (c) 2018, Los Alamos National Security, LLC
+# Produced at the Los Alamos National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+
+from spack import *
+
+
+class Eospac(Package):
+    """A collection of C routines that can be used to access the Sesame data
+       library.
+    """
+
+    homepage = "https://laws.lanl.gov/projects/data/eos.html"
+    url      = "https://laws.lanl.gov/projects/data/eos/get_file.php?package=eospac&filename=eospac_v6.4.0beta.1_r20171213193219.tgz"
+
+    version('6.4.0beta.1', 'e4e4beabf946f0b8953532832002afc2')
```

Comment:
We normally don't add alpha/beta versions of packages unless we have to. Can you add the latest stable release instead? If you really want the beta version, the stable version should be set to `preferred=True`.

Reply:
Done